compiler/rustc_middle/src/ty/util.rs

```diff
@@ -848,8 +848,8 @@ impl<'tcx> ty::TyS<'tcx> {
         }
 
         if adt_def.is_none() {
-            // Non-ADT types are not yet supported for precise tracing, so
-            // ensures they are scanned conservatively.
+            // FIXME: Non-ADT types are not yet supported for precise tracing,
+            // so ensure that they are scanned conservatively.
```

Comment:
Oops, I didn't spot a second typo -- sorry! s/so/to/

Reply:
Actually "so" is correct, what this comment is saying is that returning `(1, 0)` means the GC will treat this as a conservative block. I could reword it if that's unclear though?

Reply:
You're right!